stix2/sources/__init__.py

```diff
@@ -191,6 +256,87 @@ def query(self, query=None):
 
         """
 
+    def relationships(self, obj, relationship_type=None, source_only=False, target_only=False):
+        """Retrieve Relationships involving the given STIX object.
+
+        Only one of `source_only` and `target_only` may be `True`.
+
+        Args:
+            obj (STIX object OR dict OR str): The STIX object (or its ID) whose
+                relationships will be looked up.
+            relationship_type (str): Only retrieve Relationships of this type.
```

Comment:
Maybe add a note that if not provided, all relationships will be returned, regardless of type.